scipy/optimize/_linprog_util.py

```diff
@@ -33,13 +33,17 @@
     b_eq : 1D array, optional
         The equality constraint vector. Each element of ``A_eq @ x`` must equal
         the corresponding element of ``b_eq``.
-    bounds : sequence, optional
-        A sequence of ``(min, max)`` pairs for each element in ``x``, defining
-        the minimum and maximum values of that decision variable. Use ``None`` to
-        indicate that there is no bound. By default, bounds are ``(0, None)``
-        (all decision variables are non-negative).
-        If a single tuple ``(min, max)`` is provided, then ``min`` and
-        ``max`` will serve as bounds for all decision variables.
+    bounds : various valid formats, optional
+        The bounds of ``x``, as ``min`` and ``max`` pairs.
+        If bounds are specified for all N variables separately, valid formats are:
+        * a 2D array (2 x N or N x 2);
```

Comment:
If I said that we should accept 2 x N before, I am re-thinking it. In that case, there is ambiguity when `N == 2`. If we were to accept either 2 x N or N x 2, we'd have to have a special rule for N x 2, which probably makes things more complicated overall. I think we should only accept `N x 2`.

Reply:
> I think we should only accept N x 2.

👍

Reply:
I'll remove the 2 x N option. Might be worth a special error message.

Reply:
Please let me resolve comments when I see that the fix has been applied. This comment is not resolved because the docstring still says `* a 2D array (2 x N or N x 2);`

Reply:
OK.